package.json

```diff
@@ -19,9 +19,16 @@
 		"test": "lerna run test"
 	},
 	"devDependencies": {
-		"@amaurym/eslintrc": "^1.3.3",
+		"@types/jest": "27.0.3",
+		"eslint-config-prettier": "^8.3.0",
+		"eslint-plugin-prettier": "^4.0.0",
 		"lerna": "^3.22.1",
+		"prettier": "2.4.1",
 		"ts-jest": "^26.4.4",
 		"typescript": "^4.0.5"
+	},
+	"dependencies": {
+		"@typescript-eslint/eslint-plugin": "^4.5.0",
+		"@typescript-eslint/parser": "^4.5.0"
```

Comment:
these should probably be part of dev dep too

Reply:
done